src/sonic-yang-models/yang-templates/sonic-policer.yang.j2

```diff
@@ -0,0 +1,126 @@
+{% if yang_model_type == "cvl" %}
+/* this is sonic cvl yang model */
+{% else %}
+/* this is sonic py yang model */
+{% endif %}
+module sonic-policer {
+
+	yang-version 1.1;
+
+	namespace "http://github.com/Azure/sonic-policer";
+	prefix policer;
+
+	import sonic-types {
+		prefix stypes;
+	}
+
+	description "Policer YANG Module for SONiC OS";
+
+	revision 2022-02-03 {
+		description
+			"First Revision";
+	}
+
+	container sonic-policer {
+		container POLICER {
+			list POLICER_LIST {
+				key name;
+
+				leaf name {
+					type string;
+					description "Policer name";
+				}
+
+				leaf meter_type {
+					mandatory true;
+					type stypes:meter_type;
+					description "Policer meter type";
+				}
+
+				leaf mode {
+					mandatory true;
+                                        type stypes:policer_mode;
+					description "Policer mode";
+				}
+ 
+  				leaf color {
+                                        type stypes:policer_color_source;
+					description "Policer color Source";
+				} 
+
+				leaf cir {
+					type uint64;
+					default 0;
+					description
+						"Committed information rate for the dual-rate token
+						bucket policer.  This value represents the rate at which
+						tokens are added to the primary bucket.";
+				}
+
+				leaf cbs {
+					must "((current()/../cir) and (current()/../cir > 0))" {
+						error-message "cbs can't be configured without cir.";
+					}
+					must "(current() >= current()/../cir)" {
+						error-message "cbs must be greater than or equal to cir";
+					}
+					type uint64;
+					default 0;
+					description
+						"Committed burst size for the dual-rate token bucket
+						policer.  This value represents the depth of the token
+						bucket.";
```

Comment:
what's the unit? bytes?

Reply:
It depends on the meter type defined. The meter type can be bytes or packets. This field by itself has no units. Please refer to https://github.com/opencomputeproject/SAI/blob/16c11ee7062e630f1eaf4bb3e44d31ab58eafa5b/inc/saipolicer.h#L123